php/functions.inc.php

```diff
@@ -546,20 +546,40 @@ public function echoWeaponsTable() {
 	
 	
 	public function echoLiveRadarMapScript() {
-		echo "<br><a href='#' onclick=\"setMapCenter({lat: 42.858056, lng: 41.128056});setMapZoom(7);flushMapChanges();\">Caucasus</a> - <a href='#' onclick=\"setMapCenter({lat: 38.18638677, lng: -115.16967773});setMapZoom(7);flushMapChanges();\">Nevada</a><br>";
+		echo "<a href='#' onclick=\"map.setView([42.858056, 41.128056], 7);\">Caucasus</a> - ";
+		echo "<a href='#' onclick=\"map.setView([38.18638677, -115.16967773], 7);\">Nevada</a>";
+		echo "<br><br>";
 		
 		echo "<div id=\"map\"></div>";
-		echo "<script src=\"https://maps.googleapis.com/maps/api/js?key=AIzaSyBZoFosVL27IeHx57Wujg-v_aW3slJWItA&callback=initLiveRadarMap\"
-	        async defer></script>";
+		echo "<script src=\"https://unpkg.com/leaflet@1.9.4/dist/leaflet.js\" integrity=\"sha256-20nQCchB9co0qIjJZRGuk2/Z9VM+kNiyxNV1lvTlZBo=\" crossorigin=\"\"></script>";
+		echo "<script>initLiveRadarMap();</script>";
+		// setting markers belongs into tools.js to enable live updates live updates happen
```

Comment:
I removed this part because setting markers really belongs into tools.js, to enable dynamic updates